circuit_maintenance_parser/provider.py

```diff
@@ -172,6 +214,8 @@ class HGC(GenericProvider):
 class Lumen(GenericProvider):
     """Lumen provider custom class."""
 
+    _include_filter = {EMAIL_HEADER_SUBJECT: ["Scheduled Maintenance Window"]}
```

Comment:
Looking at more examples that I have locally, I think just "Scheduled Maintenance" would be better here. I see some subject lines like `Lumen Scheduled Maintenance #: 22194642, Scheduled` that appear to me to be valid maintenance notifications.